mender-client-acceptance-testing/Dockerfile

```diff
@@ -11,7 +11,8 @@ RUN apt-get update -qq && apt-get install -yyq python3-pip && pip3 install --upg
 # Get OS requirements from master
 RUN apt-get install -yyq wget && \
     wget https://raw.githubusercontent.com/mendersoftware/meta-mender/master/tests/acceptance/requirements_deb.txt && \
-    apt-get install -yyq $(cat requirements_deb.txt) && apt-get install -yq linux-modules-`uname -r`
+    apt-get install -yyq $(cat requirements_deb.txt | sed -e 's/libvirt-bin/libvirt-clients/') && \
+    apt-get install -yq linux-modules-`uname -r`
```

Comment:
Is `requirements_deb.txt` used anywhere else? Shouldn't we fix the original instead?

Reply:
Yes, updating the base image we can directly edit the `requirements_deb.txt` file. I suspect we might need new packages too: I have some local tweaks to make it work with Ubuntu 20.10